src/editor/components/form/input.tsx

```diff
@@ -0,0 +1,16 @@
+interface InputProps {
+  label: string,
+  value?: any,
+  disabled?: boolean,
```

Comment:
Why are these props here? The `Input` component doesn't need to know about them, that's why we have the `rest` prop.

Reply:
WebStorm